src/app/(default)/editArea/[slug]/manageTopos/components/TopoEditor.tsx

```diff
@@ -0,0 +1,126 @@
+'use client'
+import { AreaType, ClimbType } from '@/js/types'
+import { TopoClimbListSection } from './TopoClimbListSection'
+import { useEffect, useState } from 'react'
+import testImage1 from './topo2.jpg'
+import testImage2 from './topo3.jpg'
+import testImage3 from './topo.jpg'
+import { Topo } from '@/app/(default)/components/Topo/Topo'
+import { ArrowCounterClockwise, ArrowUp, Check, Circle, NumberCircleZero, Pencil } from '@phosphor-icons/react'
+import { ANCHOR, ARROW, RouteInfo, resetRouteTopo, setTerminationStyle, toggleDrawing, setActiveRoute, removeLastPoint, scaleTopo, BELAY } from '@/app/(default)/components/Topo/paperFunctions'
+import clx from 'classnames'
+import * as Slider from '@radix-ui/react-slider'
+
+export const TopoEditor: React.FC<{ area: AreaType }> = ({ area }) => {
+  const [activeRoute, setActiveRouteState] = useState<RouteInfo | undefined>()
+  const [activeTopoImage, setActiveTopoImage] = useState(testImage2)
```

Comment:
Using a typical drawing/photo editing app as an example, you may want to consider accepting image id from query string (analogous to the app opening a file).  Basically, we will have 2 separate screens:
1.  gallery view showing all photos tagged for an area (like a file explorer)
2. editor view

 
Gallery view
```
https://openbeta.io/editArea/4dec65f6-d18f-514f-8f14-ef3b5015d925/manageTopo
```

When user clicks on a photo, take them to the topo editor
```
https://openbeta.io/editArea/4dec65f6-d18f-514f-8f14-ef3b5015d925/manageTopos/editor?imageId=<image id>
```

Reply:
I'm sure I am missing something, but what would be the advantage? It seems like it would add additional navigation to editing, especially when editing multiple topos under the same area.